{"name": "test_getbyname_failure", "status": "passed", "start": 1728927427773, "stop": 1728927427774, "uuid": "8a7acfb9-f5e8-4e24-9582-6202d38d2932", "historyId": "da79b9892db5ab531f3b11a7f1273cec", "testCaseId": "da79b9892db5ab531f3b11a7f1273cec", "fullName": "Exercise.exercise_test.TestExerciseServiceWithMock#test_getbyname_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "5496-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}